### ISSUE TITLE
Multi Sessions

### ISSUE THREAD

**Issue by jalex1** [kdcz/3s_atc#44]
Not 100% sure if this is an issue but, when using multi sessions for a splash page it seems unable to open the browser to view and says it is 'waiting for product page'  even when I am using a product page url such as https://www.adidas.co.uk/nmd_r1-primeknit-shoes/BZ0223.html  
Is this just because it is not an actual splash page? Thanks

**Comment by kdcz** [comment]
I removed splash page detection in case adidas removes the current identifier so yeah it will just wait for product page identifier.

I know about the browser not showing issue will fix asap thanks for the report ;)

**Comment by Klohto** [comment]
Does this change anything for the yeezy release? How is it going to known that it passed the splash page.

**Comment by kdcz** [comment]
With the product page identifier...

**Comment by jalex1** [comment]
Thanks for the reply, I noticed however that even when the url I give the program is a product page it still says 'waiting for product page'. Does this mean that it is currently not able to recognise when it is on the product page or is this not an issue as I did not test it with a splash page?
As always keep up the great work!

**Comment by kdcz** [comment]
Which url ?

**Comment by jalex1** [comment]
Well any, I was just testing it with https://www.adidas.co.uk/mana-bounce-2.0-shoes/B39021.html for example

**Comment by kdcz** [comment]
Yeah because these "classic" product pages don't have the yeezy product page identifier.

**Comment by jalex1** [comment]
Ah I see, thanks.

**Comment by jalex1** [comment]
Just ran your new update, thanks for fixing the show browser button, great work as per. Noticed however that even when selecting to use chrome it still will run the 3atc browser.

**Comment by kdcz** [comment]
It uses 3s_atc only for size checking and sessions, I will add Chrome to sessions aswell after the yebra release asap because I was really busy recently.Atm Chrome is used only for profiles and I am sorry for that, if you can't click on the checkout button on 3s_atc I recommend you just paste the action link of the button in url bar and press enter it should work.

Pushed an update with some bug fixes.

**Comment by jalex1** [comment]
Thanks for the update. Didn't manage to cop any yebras, think just too few in stock. Hopefully will be more for next weeks release

**Comment by frasneaker** [comment]
hi man @kdcz ! Why when i select "Chrome" for Cart with browser when i pass the splash page open a 3satc browser? Is a bug? thanks for this amazing work! Appreciate it! :)

**Comment by kdcz** [comment]
Nah its not a bug Chrome is currently not implemented for splash mode,should be in next update

**Comment by jalex1** [comment]
cheers for the update, ran it today but didn't manage to cop unfortunately. Did notice that it still opens up 3s atc even what chrome is selected when running splash through guest mode though. Cheers

**Comment by kdcz** [comment]
Make sure you click the Update button after changing your settings.
